src/BuildOnWindows.cmake

```diff
@@ -0,0 +1,101 @@
+# Build on Windows
+
+set(TORCH_XPU_OPS_LIBRARIES)
+set(SYCL_LINK_LIBRARIES_KEYWORD PRIVATE)
+
+# Walk around cyclic dependence
+# libtorch_xpu.so links to libtorch_xpu_ops.a
+# libtorch_xpu_ops.a dlopens libtorch_xpu_ops_aten.so (Break cycle)
+# libtorch_xpu_ops_aten.so links to libtorch_xpu_ops_sycl_unary_binary_kernels.so and libtorch_xpu_ops_sycl_kernels.so
+# libtorch_xpu_ops_sycl_unary_binary_kernels.so and libtorch_xpu_ops_sycl_kernels.so links to libtorch_xpu.so
+add_library(
+  torch_xpu_ops
+  STATIC
+  ${ATen_XPU_CPP_SRCS}
+  "bridge.cpp")
+
+target_compile_options(torch_xpu_ops PRIVATE -DPATH_TO_TORCH_XPU_OPS_ATEN_LIB=\"torch_xpu_ops_aten.dll\")
+
+add_library(
+  torch_xpu_ops_aten
+  SHARED
+  ${ATen_XPU_NATIVE_CPP_SRCS}
+  ${ATen_XPU_GEN_SRCS})
+install(TARGETS torch_xpu_ops_aten DESTINATION "${TORCH_INSTALL_LIB_DIR}")
+target_link_libraries(torch_xpu_ops_aten PUBLIC torch_xpu)
+
+if(BUILD_SEPARATE_OPS)
+  foreach(sycl_src ${ATen_XPU_SYCL_SRCS})
+    get_filename_component(name ${sycl_src} NAME_WLE REALPATH)
+    set(sycl_lib torch-xpu-ops-sycl-${name})
+    sycl_add_library(
+      ${sycl_lib}
+      SHARED
+      SYCL_SOURCES ${sycl_src})
+    target_link_libraries(torch_xpu_ops_aten PUBLIC ${sycl_lib})
+    list(APPEND TORCH_XPU_OPS_LIBRARIES ${sycl_lib})
+
+    # Decouple with PyTorch cmake definition.
+    install(TARGETS ${sycl_lib} DESTINATION "${TORCH_INSTALL_LIB_DIR}")
+  endforeach()
+else()
+  # Split SYCL kernels into 2 libraries as categories 1) Unary+Binary 2) Others.
+  set(ATen_XPU_SYCL_UNARY_BINARY_SRCS)
+  set(ATen_XPU_SYCL_NON_UNARY_BINARY_SRCS)
+
+  foreach(sycl_src ${ATen_XPU_SYCL_SRCS})
+    string(REGEX MATCH "Binary" IS_BINARY ${sycl_src})
+    string(REGEX MATCH "Unary" IS_UNARY ${sycl_src})
+    string(REGEX MATCH "Pow" IS_POW ${sycl_src})
+    string(REGEX MATCH "Copy" IS_COPY ${sycl_src})
```

Comment:
Please add informative comments here to elaborate on why we need to take `Pow` and `Copy` as a special case.

Reply:
Will fix